### ISSUE TITLE
No support for Staff/Mace Cluster jewels

### ISSUE THREAD

**Issue by patrick-minton** [PathOfBuildingCommunity/PathOfBuilding#930]
Thank you for reporting an issue.

Before reporting a bug, **please make sure that you are running the latest version.**
You can get the latest version number from `CHANGELOG.md`

Please try to fill in as much of the template below as you're able.

* **What steps will reproduce the bug?**
import the PoB code of this character: https://poe.ninja/challengehc/builds/char/Ranggarn/RANGYELLIESTMAXIMUS
Note that it has jewels with the enchantments:

```
Added Small Passive Skills grant: Staff Attacks deal 12% increased Damage with Hits and Ailments
Added Small Passive Skills grant: Mace or Sceptre Attacks deal 12% increased Damage with Hits and Ailments
```

Note that even if you allocate those jewel skill points, the cluster wheels do not appear.

* **Does it reproduce every time? If not, why so?**
Yes

* **Additional information:**
I am pretty certain this is unique to this type of cluster jewels.

**Comment by patrick-minton** [comment]
Addendum:

I changed the enchantment to:

```
{crafted}Added Small Passive Skills grant: 10% increased Attack Damage
```

And the jewel now works correctly. Would be nice not to have to edit the jewels, though

**Comment by LocalIdentity** [comment]
Fixed in 1.4.170.1
